lib/aerospike/command/command.rb

```diff
@@ -487,6 +487,9 @@ def execute
           @node = get_node
           @conn = @node.get_connection(@policy.timeout)
         rescue => e
+          if e.is_a?(Aerospike::Exceptions::MaxConnectionsExceeded)
+            Aerospike.logger.error("Maximum connections established. No new connection can be created. #{e}")
```

Comment:
Same issue. Logging without re-raising the exception.

Reply:
Have not re-raised exception here as there is a retry loop here